assets/reports/markdown.template

```diff
@@ -0,0 +1,21 @@
+# osde2e weather report
+
+## Report Information
+This report was generated on {{.ReportDate}}
+
+## Summary
+
+{{range .Jobs}}
+* [{{.Name}}](#{{.Name}}) (Viable: {{.Viable}}){{end}}
+
+{{range .Jobs}}
```

Comment:
I feel like this is redundant. Can we only list jobs that have failures?

Reply:
I'd personally rather list all jobs -- that way we get a sense of what's actually being run as opposed to needing to know which jobs are being run. Thoughts?